test/test_local.py

```diff
@@ -34,6 +34,21 @@ def test_planes(request, run_in_tmpdir):
     diff_gold_file('plane.jou')
 
 
+@reset_openmc_ids
+def test_nested_spheres(request, run_in_tmpdir):
+    inner_sphere = openmc.Sphere(r=10.0)
+    middle_sphere = openmc.Sphere(r=20.0)
+    outer_sphere = openmc.Sphere(r=30.0)
+
+    inner_cell = openmc.Cell(region=-inner_sphere)
+    middle_cell = openmc.Cell(region=-middle_sphere)
+    outer_cell = openmc.Cell(region=-outer_sphere)
+
+    g = openmc.Geometry([outer_cell, middle_cell, inner_cell])
```

Comment:
Is the order that the cells are listed in going to have an impact? I was expecting to see the outputs in the cubit journal file ordered by cell ID, but it looks reversed.

Reply:
could also be worth tacking on a vacuum boundary condition to the outer sphere so that it makes a viable geometry that could ostensibly be run in openmc

Reply:
> Is the order that the cells are listed in going to have an impact? I was expecting to see the outputs in the cubit journal file ordered by cell ID, but it looks reversed.

I believe they'll be generated in the order they appear in the iterable passed to the Geometry object. Corresponding IDs on the volume objects in the resulting CAD would be great, but might be difficult to manage based on how Cubit handles ID spaces. An easier short-term solution would be to place each cell in it's own group with the ID in the name for reference to the original model.

> could also be worth tacking on a vacuum boundary condition to the outer sphere so that it makes a viable geometry that could ostensibly be run in openmc

Agreed that it would be nice and I'm happy to include those changes here. Right now I'm planning on spending my time covering and verifying more geometry capabilities  in the test suite (hopefully correctly in the future) than every test case being runnable in OpenMC.

Reply:
super thanks!